src/components/molecules/Visualizer/Engine/index.tsx

```diff
@@ -34,8 +34,8 @@ export type SceneProperty = {
     cameraLimitterEnabled?: boolean;
     cameraLimitterShowHelper?: boolean;
     cameraLimitterTargetArea?: Camera;
-    cameraLimitterTargetWidth: number;
-    cameraLimitterTargetLength: number;
+    cameraLimitterTargetWidth?: number;
+    cameraLimitterTargetLength?: number;
```

Comment:
cameraLimiter, not cameraLimitter ;) One too many Ts

Reply:
Can't be updated unfortunately :( , as back-end manifest is already has the id of `cameraLimitter` in some places.
I have discussed it with Baba-chan and there is no way to update this until we have a way to migrate data (so camera limiter won't break for users who already use it).